link/src/main/java/com/stripe/android/link/ui/wallet/WalletScreen.kt

```diff
@@ -0,0 +1,390 @@
+                modifier = Modifier.padding(end = 12.dp),
+                tint = Color.Unspecified
+            )
+            Text(
+                text = stringResource(id = R.string.wallet_add_new_payment_method),
+                color = MaterialTheme.colors.onPrimary,
+                style = MaterialTheme.typography.button
+            )
+        }
+    }
+}
+
+@Composable
+internal fun CardPaymentMethodItem(
+    cardDetails: ConsumerPaymentDetails.Card,
+    isSelected: Boolean,
+    onClick: () -> Unit
+) {
+    Row(
+        modifier = Modifier
+            .fillMaxWidth()
+            .height(56.dp)
+            .clickable(onClick = onClick),
+        verticalAlignment = Alignment.CenterVertically
+    ) {
+        RadioButton(
+            selected = isSelected,
+            onClick = null,
+            modifier = Modifier.padding(end = 6.dp),
+            colors = RadioButtonDefaults.colors(
+                selectedColor = MaterialTheme.colors.primary,
+                unselectedColor = MaterialTheme.colors.onBackground
+            )
+        )
+        CardDetails(card = cardDetails)
+        Spacer(modifier = Modifier.weight(1f))
+        if (cardDetails.isDefault) {
+            Box(
+                modifier = Modifier
+                    .height(20.dp)
+                    .background(
+                        color = MaterialTheme.colors.secondary,
+                        shape = MaterialTheme.shapes.small
+                    ),
+                contentAlignment = Alignment.Center
+            ) {
+                Text(
+                    text = stringResource(id = R.string.wallet_default),
+                    modifier = Modifier.padding(horizontal = 4.dp, vertical = 2.dp),
+                    color = MaterialTheme.colors.onBackground,
+                    fontSize = 12.sp,
+                    fontWeight = FontWeight.Medium
+                )
+            }
+        }
+    }
+    Divider(color = MaterialTheme.colors.onBackground, thickness = 1.dp)
+}
+
+@Composable
+internal fun PayButton(
+    label: String,
+    onButtonClick: () -> Unit
+) {
+    Box(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(vertical = 16.dp),
+        contentAlignment = Alignment.CenterEnd
+    ) {
+        TextButton(
+            onClick = onButtonClick,
+            modifier = Modifier
+                .fillMaxWidth()
+                .height(56.dp),
+            shape = MaterialTheme.shapes.medium,
+            colors = ButtonDefaults.buttonColors(
+                backgroundColor = MaterialTheme.colors.primary
+            )
+        ) {
+            Text(
+                text = label,
+                color = MaterialTheme.colors.onPrimary
+            )
+        }
+        Icon(
+            painter = painterResource(id = R.drawable.stripe_ic_lock),
+            contentDescription = null,
+            modifier = Modifier
+                .height(16.dp)
+                .width(13.dp + 40.dp)
```

Comment:
This seems a little weird. I'm assuming it has something to do with the SVG's hardcoded sizes? Maybe a comment to explain or code these values to a constant.